apps/communications/contacts/js/views/form.js

```diff
@@ -369,6 +371,36 @@ contacts.Form = (function() {
     };
   };
 
+  function showForm(event) {
+    if(event.key === 'Control') {
```

Comment:
Hm-m, maybe a dummy question, but how do you manage to get event with such key ``'Control'``?